src/Fantomas/CodePrinter.fs

```diff
@@ -52,6 +52,11 @@ type ASTContext =
           IsMemberDefinition = false
           IsInsideDotIndexed = false }
 
+let getListOrArrayExprSize ctx maxWidth xs =
```

Comment:
Could you move this function to `Context.fs` please?

Reply:
Moved